src/listeners/guildCreate.ts

```diff
@@ -25,21 +24,18 @@ export class UserEvent extends Listener {
 		if (IS_CUSTOM_BOT) {
 			// TODO: #26 Create a nice welcome message for custom bot servers
 		}
-		await createNewGuild(guild_id, inviter);
+		const _guild = await this.container.utilities.guild.get.GuildByID(guild_id);
```

Comment:
The logic currently goes as the following: 
1. Check if there is a guild entry that exists and is disabled
2. If yes, enable the guild & enable all related birthdays.
3. If no, create a new Guild.

Does this here also cover this scenario? Because on Bot leave the Guild and all related Birthdays are just disabled.